Fabric.Identity.API/scripts/Install-Identity-Utilities.psm1

```diff
@@ -776,9 +776,7 @@ function Get-SettingsFromInstallConfig {
     $tempNode = $tenantScope.SelectSingleNode($setting)
     $settingList = @()
     foreach($nodeChild in $tempNode.variable){
-        if($nodeChild.name) {
-            $settingList += $nodeChild.name
-        }
+        $settingList += @{name = $nodeChild.name; alias = $nodeChild.alias}
```

Comment:
For future reference adding objects to arrays is less performant than using lists.  Since this is performance critical, probably not a huge deal and doesn't need to be fixed.